mod/projects/set/self/project/core.haml

```diff
@@ -0,0 +1,14 @@
+%div
+  = field_nest :description
+  = nest :homepage_featured_projects, items: { view: "listing" }
+#project-tabs.tabbable
+  %ul.nav.nav-tabs
+    %li.nav-item
+      %a{class: "active nav-link", href: "#project-active", data: {toggle: "tab"}} Active
+    %li.nav-item
+      %a{class: "nav-link", href: "#project-inactive", data: {toggle: "tab"}} Inactive
+  .tab-content
+    #project-active.tab-pane.active
+      = nest "Project+Active", title: "Active Projects", items: { view: "listing" }
```

Comment:
this still has Active and Inactive hard coded.  We need to:
1. get the options from the options rule in Project+status
2. create +Project cards on each status with a +*type_plus_right rule.  Eg, Active+Projects should give you a list of all the active projects
3. make this file loop through all the statuses.